apps/orakul-ui/src/components/Scene/Scene.tsx

```diff
@@ -96,42 +102,43 @@ const Scene = ({ data, onNodeEnter, onNodeSelect, onViewChange, view }: ScenePro
     ({ source, target }: Connection) => {
       const newEdge = buildEdge(source, target);
       if (newEdge) {
-        setEdges([...edges, ]);
+        setEdges([...edges]);
       }
     },
-    [edges, setEdges],
+    [edges, setEdges]
   );
 
   const onDoubleClickHandler = useCallback(
     (_: any, node: Node) => {
       onNodeEnter && onNodeEnter(node.id);
     },
-    [onNodeEnter],
+    [onNodeEnter]
   );
 
   const highlightEdges = useCallback(
     (selectedNode: Node) => {
       setEdges((edges) =>
         edges.map((edge) => ({
           ...edge,
-          selected: edge.source === selectedNode.id || edge.target === selectedNode.id,
-        })),
+          selected:
```

Comment:
please run `npm run format` to avoid stylistic changes

Reply:
done